util/archive_builder.go

```diff
@@ -209,7 +208,9 @@ func buildSection(elements map[string]string, locatorName string) []ManifestSect
 	result := make([]ManifestSection, 0)
 	for key, value := range concatenateElementsWithSameValue(elements) {
 		manifestSectionBuilder := NewMtaManifestSectionBuilder()
-		manifestSectionBuilder.Name(key)
+		// as the key is the path to the content/artifact, write it in the manifest as base in order to normalize it
+		baseArtifactPath := filepath.Base(key)
```

Comment:
I would not do this this way because if the code changes, someone could forget to change this line. Please, make the filepath.Base(..) in the code before this function call -> that way we will ensure that the paths are "normalized"